colour/io/luts/tests/test_lut.py

```diff
@@ -461,6 +467,31 @@ def test_linear_table(self):
             self._table_3,
             decimal=7)
 
+    def test_invert(self):
+        """
+        Tests :class:`colour.io.luts.lut.LUT1D.invert`,
+        :class:`colour.io.luts.lut.LUT3x1D.invert` and
+        :class:`colour.io.luts.lut.LUT3D.invert` methods.
+        """
+        if self._LUT_factory is None:
+            return
+
+        LUT_1 = self._LUT_factory(self._table_2)
```

Comment:
Codacy found an issue: [self._LUT_factory is not callable](https://app.codacy.com/manual/colour-science/colour/pullRequest?prid=6044921)